test/com/google/javascript/jscomp/CommonJSIntegrationTest.java

```diff
@@ -59,105 +59,34 @@ public void testCrossModuleTypeAnnotation() {
             "/** @type {!Hello} */ var hello = new Hello();",
             "module.exports = Hello;"),
         LINE_JOINER.join(
-            "var module$i0 = function () {};", "var hello$$module$i0 = new module$i0();"));
+            "/** @const */ var module$i0 = {};",
+            "module$i0.default = function () {};",
+            "var hello$$module$i0 = new module$i0.default();"));
   }
 
   public void testCrossModuleTypeAnnotation2() {
     test(
         createCompilerOptions(),
         new String[] {
-          LINE_JOINER.join("/** @constructor */ function Hello() {}", "module.exports = Hello;"),
-          LINE_JOINER.join(
-              "var Hello = require('./i0');", "/** @type {!Hello} */ var hello = new Hello();")
+          "/** @constructor */ function Hello() {} module.exports = Hello;",
+          "var Hello = require('./i0'); /** @type {!Hello} */ var hello = new Hello();"
         },
         new String[] {
-          "var module$i0 = function() {};",
-          LINE_JOINER.join("var Hello = module$i0;", "var hello = new module$i0();")
+          "/** @const */ var module$i0 = {}; /** @const */ module$i0.default = /** @constructor */ function() {};",
+          "var Hello = module$i0.default; var hello = new module$i0.default();"
         });
   }
 
   public void testCrossModuleTypeAnnotation3() {
     test(
         createCompilerOptions(),
         new String[] {
-          LINE_JOINER.join("/** @constructor */ function Hello() {}", "module.exports = Hello;"),
-          LINE_JOINER.join("var Hello = require('./i0');", "/** @type {!Hello} */ var hello = 1;")
+          "/** @constructor */ function Hello() {} module.exports = Hello;",
+          "var Hello = require('./i0'); /** @type {!Hello} */ var hello = 1;"
         },
         TypeValidator.TYPE_MISMATCH_WARNING);
   }
 
-  public void testMultipleExportAssignments1() {
```

Comment:
Are we no longer supporting these use-cases?

Reply:
Well I thought not, but now I'm 2nd guessing myself. Let me go check ...

Reply:
@brad4d Those tests give errors like:

`JSC_DUP_VAR_DECLARATION. variable module$i0.default redefined, original definition at i0.js:3 at i0.js line 1 : 29 `

And that warning looks pretty correct to me. I could of course expect the warning, but I'm not sure it's really testing anything meaningful.